swap/src/monero/wallet_rpc.rs

```diff
@@ -71,8 +72,19 @@ impl WalletRpc {
                 .open(monero_wallet_rpc.tar_path())
                 .await?;
 
-            let byte_stream = reqwest::get(DOWNLOAD_URL)
-                .await?
+            let response = reqwest::get(DOWNLOAD_URL).await?;
+
+            let content_length = response.headers()[CONTENT_LENGTH]
+                .to_str()
+                .context("failed to convert content-length to string")?
+                .parse::<u64>()?;
+
+            tracing::info!(
+                "Downloading monero-wallet-rpc ({})",
```

Comment:
As far as I can see this does not provide the status...? If it does, fine. If not we should probably add something like "this can take a few minutes...".

Reply:
Correct, I shows you how big the download is. I expect the user to roughly know how long it takes to download 70MB on their internet connection.